### DISCUSSION TITLE
Basic voice bot

### DISCUSSION THREAD

**Discussion by caetanovidal** [Rapptz/discord.py#8197] [Q&A]
Hi guys, so i'm trying to implement some voice audios on my bot, and I have two questions.

Is it possible to the same class how reply to messages also play audios? and is the basic voice example in this repository working fine? I basic copy and paste that to make some tests and its not working here 
![image](https://user-images.githubusercontent.com/86735524/176768947-2447033d-7018-49a5-8b2b-cb503ab4cb7e.png)

**Comment by justanotherbyte** [comment]
Are you sure you're on the master branch of discord.py?

If not, you can install it via;
```sh
pip install git+https://github.com/Rapptz/discord.py
``` 

Note that you need `git` installed for this to work

**Reply by caetanovidal** [reply]
alright probably i'm using version 1.0, thank you

**Comment by pikaninja** [comment]
As justanotherbyte mentioned, the example in the base branch of the library is targeted at the 2.0 version, meaning you need to install it from there and not pip. If you do not want to do that, you can switch branches and use the examples there, found [here](https://github.com/Rapptz/discord.py/blob/v1.x/examples/basic_voice.py)

**Reply by caetanovidal** [reply]
alright probably i'm using version 1.0, thank you
